bundles/org.openhab.binding.semsportal/README.md

```diff
@@ -0,0 +1,45 @@
+# SEMSPortal Binding
+
+This binding can help you include statistics of your SEMS / GoodWe solar panel insallation into openHAB. It is a readonly connection that maps collected parameters to openhab channels. It provides current, day, month and total yields, as well as some income statistics if you have configured these in the SEMS portal. It requires a power station that is connected through the internet to the SEMS portal.
+
+## Supported Things
+
+There is only one thing type supported in this binding, which is the powerStation. You need to provide the identifying UUID of the solar system in your account yourself. If you have multiple powerstations that report to the same account, you can configure the same account for each powerSation and only differ the unique id of the station in each Thing.
```

Comment:
Perhaps consider a `portal` bridge thing ?!?!? Does this binding work on the kind of solar panels that have a powerStation / inverter built into each solar panel? Are there any limits to how many calls to the API/Portal you do per day?

Reply:
Yes I thought about a portal thing, but I have not found out yet what the request/response would be if you have more than 1 power station, typically, where to find the uniqueIDs of each station. The API has very limited to no documentation at all and errors are in Chinese. The requests are reverse-engineered from their own UI on the portal. I was planning to look into auto discovery later but the binding could be useful to people in its current form already because most people will have 1 station. I have several panels, they are all hooked up to 1 converter (station), so for most users 1 Thing should suffice. There is also no documentation on rate limits, but when you log in to the portal their own client does 1 minute updates so I thought a default of 5 and min of 1 minute should work.

Reply:
If I understand correctly that the call has the uniqueID in it then perhaps multiple calls are needed and a bridge would not offer anything of value.
You probably already know but the API used to be publically available and a few projects are around including this Domoticz binding...
https://github.com/dylian94/domoticz-GoodWeSEMS/blob/master/plugin.py

Also I stumbled on this which may be how to do discovery..

https://github.com/DiedB/Homey-SolarPanels/issues/28#issuecomment-521042829

Reply:
I took the calls at the Domotics binding topic to implement a Discovery Service, so you need less parameters and more things are done for you by openHAB.